src/collision/collision_system.cpp

```diff
@@ -419,6 +419,14 @@ CollisionSystem::collision_static(collision::Constraints* constraints,
   // Collision with other (static) objects.
   for (auto* static_object : m_objects)
   {
+    float static_size = static_object->get_bbox().get_width() * static_object->get_bbox().get_height();
+    float object_size = object.get_bbox().get_width() * object.get_bbox().get_height();
+    float forgiveness = 16.f * 16.f;
```

Comment:
`const float` for first two, and also `static const float FORGIVENESS` since it's an unchanging constant.